### PR TITLE
Added prettifyer command. Prettifyed tf-files. Upd changelog

### LINKED ISSUE #10: Formatting of `.tf` files [Makefile] or pre-commit hook
_originally posted in [#306](https://github.com/fredrikhgrelland/vagrant-hashistack/issues/306) by @zhenik_

Use formating 
```
terraform fmt -recursive
```

### PR DESCRIPTION
**Pull request by claesgill**
Closes #10 

* Added prettifyer command in `Makefile`
* Prettifyed all `.tf`-files

**NB! Command name (`make prettify`) is not carved in stone and is up for discussion**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,3 +54,6 @@ clean: destroy-box remove-tmp
 # helper commands
 update-box:
 	@SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} vagrant box update || (echo '\n\nIf you get an SSL error you might be behind a transparent proxy. \nMore info https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md#proxy\n\n' && exit 2)
+
+prettify:
+	@terraform fmt -recursive && echo Trying to prettify all .tf files. Do  \'git status\' to see which files got prettified.
```
Comment (pdmthorsrud) [inline Makefile]: Using `git status` to see what's been prettified can be misleading. I imagine people are working when using this command, in which case `git status` will show much more than what prettify necessarily have touched. Does the format-command possibly give any feedback on changed files that we could use?

Comment (claesgill) [reply Makefile]: Yeah, you're right. The format-command does not give any feedback unfortunately. Do you think I should remove the whole echo or should we provide a more descriptive text?

Comment (pdmthorsrud) [reply Makefile]: Damn. Maybe just remove the git status part?

Comment (claesgill) [reply Makefile]: Yeah, let's just do that for now

**Review by pdmthorsrud — APPROVED** [review]

**Review by zhenik — APPROVED** [review]

**Comment by claesgill** [comment]
Before I merge this one, can we clearify [THIS](https://github.com/fredrikhgrelland/vagrant-hashistack-template/issues/10#issuecomment-694877624) question first, @zhenik @pdmthorsrud ?

**Comment by zhenik** [comment]
> Before I merge this one, can we clearify [THIS](https://github.com/fredrikhgrelland/vagrant-hashistack-template/issues/10#issuecomment-694877624) question first, @zhenik @pdmthorsrud ?

What are the options to automate pre-commit procedures? Does it worth it?

E.g. options: 
- integrate with IDEA
- build sh script, run before commit
- leave it manual (delegate to contributors), just update documentation of commit's requirement for code contribution

**Comment by fredrikhgrelland** [comment]
I personally like something like `make pre-commit` or something

**Comment by pdmthorsrud** [comment]
I wouldn't like it to be something that is forced upon users (the tests when creating PR does that), so I agree with `make pre-commit`

**Comment by fredrikhgrelland** [comment]
Anything that requires installing in my env is bad. If it is transparently included in the box I am fine with anything. But KISS..

**Comment by claesgill** [comment]
I agree with both @pdmthorsrud and @fredrikhgrelland, and think that `make pre-commit` sounds like a good idea.
Should I merge this PR and create a new issue regarding this?

**Comment by fredrikhgrelland** [comment]
Your Choice I guess

**Comment by claesgill** [comment]
> Your Choice I guess

Okey, I'll merge and create another issue then.

Edit: 
New issue is #24

**Pull request merged**